conda-envs/mlos.yml

```diff
@@ -15,14 +15,13 @@ dependencies:
   - jupyter
   - ipykernel
   - nb_conda_kernels
-  - matplotlib
+  - matplotlib-base
```

Comment:
Does this still allow plotting to work correctly?

Reply:
yes, I've just checked it locally. There was a weird unit test failure on the first run after I had rebuilt the conda env:
`FAILED mlos_viz/mlos_viz/tests/test_dabl_plot.py::test_dabl_plot - _tkinter.TclError: couldn't read file "C:/Users/sergiym/AppData/Local/anaconda3/envs/mlos/Library/lib/tcl8.6/auto.tcl": no such file or directory`
when I checked, the tcl file was there and I could not reproduce the failure again. Maybe this file is generated the first time matplotlib is used and there was some race condition when I first ran the unit tests.